--- rekonq-2.1/src/data/defaultbookmarks.xbel.bookmark~	2013-01-27 20:50:51.751973739 +0100
+++ rekonq-2.1/src/data/defaultbookmarks.xbel	2013-01-27 21:11:24.069799431 +0100
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE xbel>
 <xbel version="1.0">
-    <bookmark href="http://qt-project.org/">
-        <title>Qt site</title>
+    <bookmark href="/usr/share/doc/HTML/index.html">
+	<title>OpenMandriva Linux</title>
     </bookmark>
-    <bookmark href="http://webkit.org/">
-        <title>WebKit.org</title>
+    <bookmark href="http://www.openmandriva.org/">
+        <title>OpenMandriva</title>
     </bookmark>
-    <bookmark href="http://rekonq.kde.org/">
-        <title>rekonq site</title>
+    <bookmark href="http://forums.openmandriva.org/">
+	<title>OM Forum</title>
     </bookmark>
-    <bookmark href="http://kde-apps.org/">
-        <title>KDE-Apps.org</title>
+    <bookmark href="http://wiki.openmandriva.org/">
+	<title>OM Wiki</title>
     </bookmark>
-    <bookmark href="http://www.kde.org/">
-        <title>KDE site</title>
+    <bookmark href="http://rekonq.kde.org/">
+        <title>rekonq site</title>
     </bookmark>
 </xbel>
