--- rekonq-2.0/src/rekonq.kcfg.preview~	2012-12-28 19:05:24.000000000 +0100
+++ rekonq-2.0/src/rekonq.kcfg	2012-12-29 18:07:19.592383938 +0100
@@ -63,10 +63,10 @@
 <!-- New Tab Page Settings -->
 <group name="NewTabPage">
     <entry name="previewNames" type="StringList">
-        <default>KDE Homepage,rekonq site</default>
+        <default>Mandriva Linux,Mandriva Blog,rekonq site</default>
     </entry>
     <entry name="previewUrls" type="StringList">
-        <default>http://www.kde.org/,http://rekonq.kde.org/</default>
+	    <default>http://start.stage.mandriva.com/,http://blog.mandriva.com/,http://rekonq.kde.org/</default>
     </entry>
 </group>
 
@@ -80,7 +80,7 @@
         <default>false</default>
     </entry>
     <entry name="homePage" type="String">
-        <default>http://www.kde.org/</default>
+	    <default>file:/usr/share/doc/HTML/index.html</default>
     </entry>
     <entry name="downloadPath" type="Url">
         <default code="true">KUrl(KGlobalSettings::downloadPath())</default>
